src/parser/jobs/brd/PitchPerfect.js

```diff
@@ -0,0 +1,253 @@
+		strenght: 0.15,
+	},
+	{
+		id: STATUSES.CHAIN_STRATAGEM.id,
+		strenght: 0.15,
+	},
+	{
+		id: STATUSES.CRITICAL_UP.id,
+		strenght: 0.02,
+	},
+	{
+		id: STATUSES.THE_SPEAR.id,
+		//fuck royal road
+		strenght: 0.10,
+	},
+	{
+		id: STATUSES.STRAIGHT_SHOT.id,
+		strenght: 0.10,
+	},
+]
+
+const DHIT_MOD = 1.25
+
+const DISEMBOWEL_STRENGTH = 0.05
+const TRAIT_STRENGTH = 0.20
+
+const DEVIATION_PRECISION = 3
+
+export default class PitchPerfect extends Module {
+	static handle = 'pitchPerfect'
+	static dependencies = [
+		'additionalEvents', // eslint-disable-line xivanalysis/no-unused-dependencies
+		'combatants',
+		// Relying on the normaliser from this for the hit type fields
+		'hitType', // eslint-disable-line xivanalysis/no-unused-dependencies
+	]
+
+	// Represents a map of IDs and statuses for each enemy in this parse
+	_enemies = {}
+
+	// Represents the player statuses
+	_player = {
+		statuses: {},
+		hasStatus(statusId) { return this.statuses[statusId] || false },
+	}
+
+	normalise(events) {
+
+		const damageInstances = []
+		const ppEvents = []
+
+		for (const event of events) {
+			// Registers buffs/debuffs statuses on the respective entity (either player or enemies)
+			if (event.type.startsWith('apply') || event.type.startsWith('remove')) {
+				if (event.targetID === this.combatants.selected.id && event.ability) {
+					this._player.statuses[event.ability.guid] = event.type.startsWith('apply')
+				} else if (!event.targetIsFriendly) {
+					const enemy = this._getEnemy(event.targetID)
+					enemy.statuses[event.ability.guid] = event.type.startsWith('apply')
+				}
+
+			// For every damage event that:
+			// - comes from the player
+			// - is not a dot tick
+			} else if (
+				event.type === 'damage'
+				&& event.sourceID === this.combatants.selected.id
+				&& event.ability
+				&& !event.tick
+			) {
+				// Fixing the multiplier
+				let fixedMultiplier = event.debugMultiplier
+				if (
+					event.ability.guid !== ACTIONS.THE_WANDERERS_MINUET.id
+					&& event.ability.guid !== ACTIONS.MAGES_BALLAD.id
+					&& event.ability.guid !== ACTIONS.ARMYS_PAEON.id
+				) {
+					// Band-aid fix for disembowel (why, oh, why)
+					if (this._getEnemy(event.targetID).hasStatus(STATUSES.PIERCING_RESISTANCE_DOWN.id)) {
+						fixedMultiplier = Math.trunc((fixedMultiplier + DISEMBOWEL_STRENGTH) * 100) / 100
+					}
+					// AND ALSO FOR RANGED TRAIT, BECAUSE APPARENTLY IT'S PHYSICAL DAMAGE ONLY REEEEEEEEEE
+					fixedMultiplier = Math.trunc((fixedMultiplier + TRAIT_STRENGTH) * 100) / 100
+				}
+
+				// If it's a Pitch Perfect damage
+				if (event.ability.guid === ACTIONS.PITCH_PERFECT.id) {
+
+					ppEvents.push({event: event, rawDamage: event.amount / fixedMultiplier})
+
+				// Otherwise, if it doesn't have a conditional potency (Sidewinder and Pitch Perfect), it will be used to calculate 'K'
+				} else if (event.ability.guid !== ACTIONS.SIDEWINDER.id) {
+
+					// ...let's not count Spears for now
```

Comment:
`core/Arcanum` should be adding a strength modifier as part of normalisation, if that's what's holding this up.